packages/preview/koma-labeling/0.2.0/CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+# [unreleased](https://github.com/kamasylvia/koma-labeling/releases/tag/unreleased)
```

Comment:
It would be nice to fill the changelog (or to just delete the file if you don't actually plan to use it).